### PR TITLE
Content creation should be done on prod (for now) but should be restr…

### PR DESCRIPTION
**Pull request by mehalshah**
…icted to those who can edit courses (aka admins)

### CODE REVIEW

File: dashboard/app/controllers/plc/content_creator_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Plc::ContentCreatorController < ApplicationController
-  before_filter :require_levelbuilder_mode
-
   def show_courses_and_modules
+    authorize! :read, Plc::Course
```
Comment (aoby) [inline dashboard/app/controllers/plc/content_creator_controller.rb]: The description says restricted to users who can _edit_ courses. That should be authorize! :manage. This is read access. Either the description is wrong or this is too permissive.

Comment (mehalshah) [inline dashboard/app/controllers/plc/content_creator_controller.rb]: Admins can read/write courses, no one else can do anything to them right now. But seeing as how that will eventually change, I'll adjust to :manage

Comment (aoby) [inline dashboard/app/controllers/plc/content_creator_controller.rb]: Actually I see that you don't have any specific permissions set for Plc::Course, so admin can read and manage and this is effectively the same thing.

Comment (mehalshah) [inline dashboard/app/controllers/plc/content_creator_controller.rb]: I'll switch to :manage for future usage

**Comment by aoby** [comment]
lgtm with the above change

**Pull request merged**